src/utils/error/error_utils.go

```diff
@@ -0,0 +1,56 @@
+package error
+
+import (
+	"fmt"
+	"net/http"
+)
+
+type e struct {
+	ErrorCode    int    `json:"code"`
+	ErrorMessage string `json:"message"`
+}
+
+type RestError interface {
+	Error() string
+	Code() int
+	Message() string
+}
+
+//Return custom error message
```

Comment:
By convention, all the descriptions should start with the name of the struct, interface, function, etc name
Please change all the descriptions